os-eap-txn-partition/added/partitionPV.sh

```diff
@@ -0,0 +1,202 @@
+source ${JBOSS_HOME}/bin/launch/openshift-node-name.sh
```

Comment:
Shouldn't these be rolled into the existing partitionPV.sh script?  My concern is that we're diverging and will have to start maintaining these scripts in parallel.

Reply:
do you mean the `os-eap-txn-partition` vs. `os-partition` module, right? That's what I was pretty unsure how to do so. The `os-partition` module is not used only for the txn recovery but for message migration too. But my concern and testing was the txn recovery. If you think I can try to merge those two modules to one but still there will need to be (probably?) two kinds of algorithms - one for `split-lock` approached used in messaging case, the other for `txn recovery`. Or if you will I can try to remove the `split-lock` approach completely and replace with the `txn api query` approach. Just currently I'm not sure what issues could turn up.

Reply:
I thought the two were delegating the recovery work and other app specific details.  This script it sourced by the product scripts, which provide implementations for `runMigration()`.  I would expect the same paradigm to be used.  I believe the basic script implements a strategy pattern (e.g. split the dir, add markers, cleanup, etc.), where certain steps have product specific implementations (e.g. `runMigration()`).  bash is a bit of a pita for this, but I think that's why we're sourcing the file: so the consumer can provide implementations for the stubs.

Reply:
I see. Just I'm sorry for that but I do not follow what should be changed in my case. I was thinking in the similar way (even not specified such precisely as you did). There is a new module `os-eap-txn-partition` which provides implementation of the migration itself. The root scripts (which calls the `runMigration()`) are untouched (...ok, that's not true as I needed a way to provide a bit different parameters than it was originally intended. Still I was trying to not touch any original behavior and just adding a line to get my implementation working)